packages/itwinui-css/src/bridge.scss

```diff
@@ -201,12 +216,18 @@
   }
 
   /* ---------------------------------------------------------------------------- */
-  /* DropdownMenu */
+  /* ComboBox */
 
-  .iui-menu {
-    // StrataKit uses different surface colors for the dropdowns
-    background-color: var(--ids-color-bg-elevation-level-1);
```

Comment:
This needs to be added back. Currently the menu looks wrong. It should be `bg-elevation-level-1`, not `bg-page-base`.

Reply:
Dialog also looks wrong. It should be `bg-elevation-base`.

And Popover surface should be `bg-elevation-level-1`.